x-pack/platform/packages/shared/kbn-inference-endpoint-ui-common/src/components/providers/selectable.tsx

```diff
@@ -81,11 +90,31 @@ export const SelectableProvider: React.FC<SelectableProviderProps> = ({
   const EuiSelectableContent = useCallback<NonNullable<EuiSelectableProps['children']>>(
     (list, search) => (
       <>
-        {search}
+        <EuiFlexGroup gutterSize="xs" alignItems="center">
+          <EuiFlexItem>{search}</EuiFlexItem>
+          {currentSolution && Object.keys(solutionKeys).includes(currentSolution) ? (
+            <EuiFlexItem grow={false}>
+              <EuiButtonGroup
+                buttonSize="s"
+                legend="Solution filter"
```

Comment:
Does this need to be internationalized?

Reply:
Yeah - updated in aac7542c8976c1d454bd8ae0cef26293a2ab9166